spec/mailers/task_mailer_spec.rb

```diff
@@ -0,0 +1,32 @@
+require "rails_helper"
+
+RSpec.describe TaskMailer, type: :mailer do
+  fixtures :tasks, :responsibilities
+
+  describe "when task has turns" do
+
+    let(:task)  { tasks(:weekly) }
+
+    it "should build the email" do
+      email = TaskMailer.notification(task)
+
+      expect(email.to).to match_array(task.turns.first.emails)
+      expect(email.cc).to include(task.email)
+      expect(email.subject).to include(task.turns.first.groups.first.name)
+      expect(email.text_part.body.to_s).to include(task.turns.first.groups.first.name)
```

Comment:
Sure! I just adapted the example from http://guides.rubyonrails.org/testing.html#testing-your-mailers using minitest

I am changing it